samples/bootloader/README.rst

```diff
@@ -69,6 +69,11 @@ By default, the bootloader sample will automatically generate and provision publ
 Alternatively, to facilitate the manufacturing process of a device with the bootloader sample, it is possible to decouple this process and program the sample HEX file and the HEX file containing the public key hashes separately.
 If you choose to do so, use the Python scripts in ``scripts\bootloader`` to create and provision the keys manually.
 
+   .. note::
+      On nRF9160, the provisioning data is held in the OTP region in UICR.
+      Because of this, the UICR must be cleared when flashing the bootlaoder.
+      This can be done by calling ``west flash`` with the ``--erase`` option.
```

Comment:
How do you do it in SES?

Reply:
:upside_down_face:

Reply:
@ru-fu: I added a sentence about SES, and added a bit more text as well.